crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -64,6 +64,24 @@ def services
     end
   end
 
+  api :POST, "/api/upgrade/controllers", "Orchestrate the upgrade of controller nodes"
```

Comment:
Well, user can't choose which controller to upgrade anyway.

This change I'm proposing should IMHO make it better for the UI team, because they want to show "upgrading node XY" while this is being done. So if the API would be created the way to only upgrade one controller at the time, UI would know better how to show the progress.

Other option (which is currently proposed in this PR) is to have API call for all controllers.